_includes/footer.html

```diff
@@ -1,66 +1,169 @@
-        <div class="usa-identifier__container">
-            <div class="usa-identifier__logos">
-                <a href="https://www.gsa.gov">
-                    {% image_with_class "_img/gsa-logo.svg" "usa-identifier__logo" "GSA logo identifier" %}
-                </a>
+    <div class="footer-section-bottom usa-footer__primary-section bg-accent-warm-light border-top border-accent-warm-dark">
+        <div class="grid-container  padding-bottom-4">
+            <div class="grid-row grid-gap-4">
+                <div class="mobile-lg:grid-col-6 desktop:grid-col-3">
+                    <section class="usa-footer__primary-content usa-footer__primary-content--collapsible">
+                        <h4 class="usa-footer__primary-link">Get in touch</h4>
+                        <ul class="usa-list usa-list--unstyled">
+                            <li class="usa-footer__secondary-link"><a href="/contact">Customer support</a></li>
+                            <li class="usa-footer__secondary-link"><a href="mailto:inquiries@cloud.gov">Business
+                                inquiries</a></li>
+                            <li class="usa-footer__secondary-link"><a href="https://join.tts.gsa.gov/">Join our team</a>
+                            </li>
+                        </ul>
+                    </section>
+                </div>
+                <div class="mobile-lg:grid-col-6 desktop:grid-col-3">
+                    <section class="usa-footer__primary-content usa-footer__primary-content--collapsible">
+                        <h4 class="usa-footer__primary-link">For developers</h4>
+                        <ul class="usa-list usa-list--unstyled">
+                            <li class="usa-footer__secondary-link"><a
+                                    href="/docs/ops/repos/#repositories">Repositories</a></li>
+                            <li class="usa-footer__secondary-link"><a href="/docs/pricing/free-limited-sandbox/">Free
+                                developer sandbox</a></li>
+                            <li class="usa-footer__secondary-link"><a href="https://cloudgov.statuspage.io/">cloud.gov
+                                status</a></li>
+                        </ul>
+                    </section>
+                </div>
+                <div class="mobile-lg:grid-col-6 desktop:grid-col-3">
+                    <section class="usa-footer__primary-content usa-footer__primary-content--collapsible">
+                        <h4 class="usa-footer__primary-link">Policies</h4>
+                        <ul class="usa-list usa-list--unstyled">
+                            <li class="usa-footer__secondary-link"><a
+                                    href="https://www.gsa.gov/vulnerability-disclosure-policy">Vulnerability disclosure
+                                policy</a></li>
+                            <li class="usa-footer__secondary-link"><a href="https://18f.gsa.gov/open-source-policy/">Open
```

Comment:
it's long-since time we made our own copy of this open source policy for our own site and linked to it instead. paging @wz-gsa to make sure we're allowed to do that. I'm so tired of linking to 18Fs stuff when we've been our own program for years now.